database/migrations/2019_05_06_171743_add_company_id_to_consumables_users.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddCompanyIdToConsumablesUsers extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('consumables_users', function ($table) {
+            $table->integer('company_id')->nullable();
```

Comment:
We should add `->default(null)` here too, because different configs of MySQL can behave differently.